shared/src/netlink.rs

```diff
@@ -43,14 +46,31 @@ fn netlink_call(
         ));
     }
 
-    let n_received = socket.recv(&mut buf[..], 0).unwrap();
-    let response = NetlinkMessage::<RtnlMessage>::deserialize(&buf[..n_received])
-        .map_err(|e| io::Error::new(io::ErrorKind::InvalidData, e))?;
-    log::trace!("netlink response: {:?}", response);
-    if let NetlinkPayload::Error(e) = response.payload {
-        return Err(e.to_io());
+    let mut responses = vec![];
+    loop {
+        let n_received = socket.recv(&mut buf[..], 0).unwrap();
```

Comment:
do we want to keep the `.unwrap()`s on netlink socket, or shall we convert them to `?`? I guess IO errors don't really happen on the netlink socker, but still.

Reply:
Ah yep, it sould be `?`.